### ISSUE TITLE
nspawn: Personality for armv7

### ISSUE THREAD

**Issue by bgamari** [systemd/systemd#7307]
On x86 `systemd-nspawn` supports the `x86` and `x86-64` personalities, allowing one to run a 32-bit container in a 64-bit environment. The same thing should be possible with `arm` and `aarch64`.

### Submission type

 - Request for enhancement (RFE)

### systemd version the issue has been seen with

232

### Used distribution

Debian on AArch64 hardware

### Expected behaviour you didn't see

```
$ systemd-nspawn --personality arm
[it just works]
```

### In case of bug report: Unexpected behaviour you saw

```
$ systemd-nspawn --personality arm
Unknown or unsupported personality 'arm'.
```

**Comment by bgamari** [comment]
Looking at the implementation, it actually seems that this might have been fixed in b7cac52779d52eff6ed03103922ffe1208a7baa9. I'll need to upgrade my test environment to verify.

See also #4862.

**Comment by bgamari** [comment]
Indeed this appears to be fixed.
